examples/logging/jul/README.md

```diff
@@ -0,0 +1,7 @@
+JUL Example
+---
+
+This example shows how to use Java Util Logging with MDC
+ using Helidon API.
+ 
+The example can be built using GraalVM native image as well.
```

Comment:
I was thinking if you don't want to add exact commands how to build this example as we do have in our other examples. And possibly also command for native-image build since you are explicitly mentioning it.

Reply:
Fixed